addons/mail/static/src/components/channel_member_list/channel_member_list.xml

```diff
@@ -4,21 +4,15 @@
     <t t-name="mail.ChannelMemberList" owl="1">
         <t t-if="channelMemberListView">
             <div class="o_ChannelMemberList d-flex flex-column overflow-auto bg-light" t-attf-class="{{ className }}" t-ref="root">
-                <t t-if="channelMemberListView.onlineCategoryView">
-                    <ChannelMemberListCategory record="channelMemberListView.onlineCategoryView" category="'online'" channelMemberListView="channelMemberListView"/>
+                <t t-if="channelMemberListView.channel.channelMembers.length gt 0">
```

Comment:
unreadable, would move condition to computed field

Reply:
Why those changes in this PR though? This does a lot more than just "introduce Channel and ChannelPartner model" as the title would suggest